commercial/app/controllers/commercial/JobsController.scala

```diff
@@ -31,7 +31,9 @@ class JobsController(jobsAgent: JobsAgent) extends Controller with implicits.Req
 
   def renderJobs = Action { implicit request =>
     jobSample(specificIds, segment) match {
-      case Nil => NoCache(jsonFormat.nilResult.result)
+      case Nil => Cached(60.seconds) {
+        jsonFormat.nilResult
+      }
```

Comment:
Don't change it but you can write this in one line. ie: `case Nil => Cached(60)(jsonFormat.nilResult)`